src/NorthwindWebsite/API/Controllers/ProductsController.cs

```diff
@@ -23,4 +23,74 @@ public async Task<ActionResult<List<Product>>> GetProducts()
 
         return products.Products;
     }
+
+    [HttpGet("{id}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
```

Comment:
If I am not mistaking, if you are returning `ActionResult<T>`, you don't need to specify it explicitly, `T` will be automatically recognised as returned type.